charts/stable/unifi/Chart.yaml

```
@@ -22,7 +22,7 @@ sources:
   - https://github.com/jacobalberty/unifi-docker
   - https://unifi-network.ui.com
 type: application
-version: 15.0.11
+version: 15.0.12
```

```suggestion
version: 16.0.0
```

Going to assume potentially breaking.